src/LocalAgent/Dockerfile

```diff
@@ -1,28 +1,31 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-cbl-mariner2.0 AS base
 WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0 AS build
 WORKDIR /src
-COPY ["nuget.config", "."]
-COPY ["LocalAgent/localagent.csproj", "LocalAgent/"]
-COPY ["library/library.csproj", "library/"]
-COPY ["common/common.csproj", "common/"]
-COPY ["common.json/common.json.csproj", "common.json/"]
-RUN dotnet restore "LocalAgent/localagent.csproj"
+
+COPY ["/src/nuget.config", "."]
+COPY ["/src/LocalAgent/LocalAgent.csproj", "LocalAgent/"]
```

Comment:
💡 Observation/thoughts 🥷☕️: if we have `workingdir` already mentioned as `/src` then I think `COPY ...` will route through `/src` https://docs.docker.com/engine/reference/builder/#workdir